enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1414,7 +1414,8 @@ func (p *PebbleCache) blobDir() string {
 }
 
 func (p *PebbleCache) lookupFileMetadataAndVersion(ctx context.Context, iter pebble.Iterator, key filestore.PebbleKey) (*rfpb.FileMetadata, filestore.PebbleKeyVersion, error) {
-	ctx, spn := tracing.StartSpan(ctx)
+	// Skip lint to ensure the correct ctx is used if it's needed.
+	ctx, spn := tracing.StartSpan(ctx) //nolint:SA4006
```

Comment:
I don't know how you feel about this, but one way we could fix this is by changing the arg name from ctx to rawCtx or something else, then code would be:

ctx, spn := tracing.StartSpan(rawCtx)

This would remove the nead to have `//nolint:SA4006` everywhere.

Reply:
SGTM